pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -257,9 +259,36 @@ func convertToEnvoyJwtConfig(jwtRules []*v1beta1.JWTRule) *envoy_jwt.JwtAuthenti
 			})
 		}
 		provider.FromParams = jwtRule.FromParams
-
 		jwtPubKey := jwtRule.Jwks
-		if jwtPubKey == "" {
+		isMeshCluster := false
+		direction, _, host, port := model.ParseSubsetKey(jwtRule.Issuer)
```

Comment:
So the user would actually put `issuer: outbound|8080|foo|bar.svc`? This seems like its not the right API, as its leaking impl details of cluster name into a first class API.

Reply:
I think based on our discussion in the issue, this should be the normal URL and not the cluster name.

Reply:
How would that normal url  look like? May be I missed that point. Remember Envoy just needs a cluster name.